services/kuber/server/domain/usecases/remove_lb_scrape_config.go

```diff
@@ -0,0 +1,34 @@
+package usecases
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/berops/claudie/internal/utils"
+	"github.com/berops/claudie/proto/pb"
+	"github.com/berops/claudie/services/kuber/server/domain/utils/lbScrapeConfig"
+)
+
+// RemoveLBScrapeConfig deletes the Kubernetes secret containing Prometheus scrape config related to
+// the LB clusters attached to given K8s cluster.
+// TODO: understand when this will occur.
```

Comment:
If LB is detached, the scrape config should be cleaned up from the managed cluster